conf/cmi/block.block.ibmchatapp.yml

```diff
@@ -26,4 +26,4 @@ visibility:
+    pages: "<front>\r\n/asiakkaan-tiedot-ja-oikeudet\r\n/asiakkaan-tiedot-ja-oikeudet/*\r\n/sosiaalinen-ja-taloudellinen-tuki\r\n/sosiaalinen-ja-taloudellinen-tuki/*\r\n/vammaisten-palvelut\r\n/vammaisten-palvelut/*\r\n/toihin-meille\r\n/toihin-meille/*\r\n/terveydenhoito/kuntoutus-ja-terapiat/puheterapia/aikuisten-puheterapia\r\n/terveydenhoito/kuntoutus-ja-terapiat/kuntoutussuunnittelu\r\n/terveydenhoito/kuntoutus-ja-terapiat/kuntoutusohjaus\r\n/halsovard/rehabilitering-och-terapi/talterapi/logopedi-for-vuxna\r\n/halsovard/rehabilitering-och-terapi/rehabiliteringsplanering\r\n/halsovard/rehabilitering-och-terapi/rehabiliteringshandledning\r\n/health-care/rehabilitation-and-therapies/speech-therapy/adult-speech-therapy\r\n/health-care/rehabilitation-and-therapies/rehabilitation-planning\r\n/health-care/rehabilitation-and-therapies/rehabilitation-counselling\r\n/terveydenhoito/terveysasemat\r\n/terveydenhoito/terveysasemat/*\r\n/halsovard/halsostationer\r\n/halsovard/halsostationer/*\r\n/health-care/health-stations\r\n/health-care/health-stations/*\r\n/terveydenhoito/sairaalat-ja-poliklinikat\r\n/halsovard/sjukhus-och-polikliniker\r\n/health-care/hospitals-and-outpatient-clinics\r\n/terveydenhoito/sairaalat-ja-poliklinikat/*\r\n/halsovard/sjukhus-och-polikliniker/*\r\n/health-care/hospitals-and-outpatient-clinics/*\r\n/terveydenhoito/hammashoito\r\n/terveydenhoito/hammashoito/*\r\n/halsovard/tandvard\r\n/halsovard/tandvard/*\r\n/health-care/dental-care\r\n/health-care/dental-care/*\r\n/terveydenhoito/kiireellinen-hoito-ja-paivystys\r\n/halsovard/bradskande-vard-och-halsocentralsjour\r\n/health-care/urgent-and-emergency-care\r\n/terveydenhoito/kiireellinen-hoito-ja-paivystys/*\r\n/halsovard/bradskande-vard-och-halsocentralsjour/*\r\n/health-care/urgent-and-emergency-care/*"
```

Comment:
This block has the page visibility conditions negated, so it's displayed on all pages except these. To add it to some pages we need to remove the URLs from this list.